docs/contents/dev-docs/testing.md

```diff
@@ -1,17 +1,10 @@
 # Testing
 
-Welcome to the ACK developer preview! In the following, we will take you
-through the steps to test ACK for the currently supported AWS services:
-
-- Amazon ECR
-- Amazon S3
-- Amazon SNS
-- Amazon API Gateway V2
-- Amazon DynamoDB
-- [Amazon ElastiCache](https://github.com/aws/aws-controllers-k8s/tree/main/services/elasticache)
-- AWS Step Functions
-
-If you run into any problems when testing one of the above services,
+In the following, we will take you through the steps to run end-to-end (e2e)
+tests for the ACK service controller for S3. You may use these steps to run e2e
+tests for other ACK service controllers.
+
+If you run into any problems when testing a service controller, please
 [raise an issue](https://github.com/aws/aws-controllers-k8s/issues/new/choose)
 with the details so we can reproduce your issue.
```

Comment:
Should this link now be `https://github.com/aws-controllers-k8s/community/issues/new/choose`?

Reply:
nice catch. will update.